app/views/admin/responses/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_for [:admin, consultation, consultation_response], url: [:admin, consultation, consultation_response.singular_routing_symbol] do |form| %>
+  <%= render "govuk_publishing_components/components/fieldset", {
+    legend_text: "Published on",
+    heading_size: "s"
```

Comment:
I think this should be "l" s it matches the govspeak editor input size

Reply:
Yes, wasn't sure about that from a like-for-like point of view but probably makes more sense 👍